Apps/AdminWebClient/src/ClientApp/src/plugins/inversify.config.ts

```diff
@@ -49,7 +49,7 @@ container
   .inSingletonScope();
 container
   .bind<ICommunicationService>(SERVICE_IDENTIFIER.CommunicationService)
-  .to(MockCommunicationService)
+    .to(RestCommunicationService)
```

Comment:
looks like you are having some spacing issues again, running npm run lintfix should fix this

Reply:
Thanks @tiagograf-AOT, I have just committed some changes to fix this.